shared/metrics/src/main/java/io/pravega/shared/metrics/DynamicLoggerImpl.java

```diff
@@ -9,7 +9,7 @@
  */
 package io.pravega.shared.metrics;
 
-import com.codahale.metrics.MetricRegistry;
+import io.micrometer.core.instrument.MeterRegistry;
```

Comment:
Did we completely remove `codehale`? If so, shall we remove the gradle dependency?

Reply:
Yes, it will be removed eventually.
I need the old libraries for the moment to compare Micrometer with Dropwizard.

Reply:
Please remove this from gradle. You can run your benchmark against this branch but both of these should not make it to master.

Reply:
Now codehal (and readytalk) are removed completely.